src/GroupRow.tsx

```diff
@@ -42,43 +24,47 @@ const groupRow = css`
 
 const groupRowClassname = `rdg-group-row ${groupRow}`;
 
+interface GroupRowRendererProps<R, SR> extends BaseRenderRowProps<R, SR> {
+  row: GroupRow<R>;
+  toggleGroup: (expandedGroupId: unknown) => void;
+}
+
 function GroupedRow<R, SR>({
-  id,
-  groupKey,
-  viewportColumns,
-  childRows,
-  rowIdx,
+  className,
   row,
-  gridRowStart,
-  height,
-  level,
-  isExpanded,
+  rowIdx,
+  viewportColumns,
   selectedCellIdx,
+  lastFrozenColumnIndex,
```

Comment:
do we need this prop?

Reply:
Good catch. Does not look like it is needed on the group row